api/src/main/java/io/druid/data/input/impl/PrefetchableTextFilesFirehoseFactory.java

```diff
@@ -57,20 +58,20 @@
  * by this class provides three key functionalities.
  *
  * <ul>
```

Comment:
We don't publish Javadocs, so I would optimize them for reading from IDE, not rendered. E. g. formatting lists with indentation and `-` (dashes), instead of `<li></li>`

Reply:
Makes sense. Replaced tags with dashes. Also added some ```<br/>``` tags for better rendering because at least I sometimes read in a rendered form.